app/code/core/Mage/Core/Model/Email/Template/Abstract.php

```diff
@@ -117,7 +117,7 @@ protected function _getLogoUrl($store)
                 return Mage::getBaseUrl('media') . $uploadDir . '/' . $fileName;
             }
         }
-        return Mage::getDesign()->getSkinUrl('images/logo_email.gif');
+        return Mage::getDesign()->getSkinUrl('images/logo.svg');
```

Comment:
mmm as always you have a great point. would it be possible if the user has a custom theme with a custom logo? I wouldn't want to be stuck with a gif logo forever but it's still can't force the users to migrate to svg.

Reply:
Something like the `??` operator? If SVG does not exist, fallback to GIF.

Reply:
I believe that some clients, GMail in particular, don't support SVG at all.

Reply:
mmmm https://designmodo.com/html-css-emails I guess I'll have to revert the change regarding the email part and leave it as it was before...

Reply:
It's truly a shame we can't use SVG in email. One option though, is that we could modify kiatng's code to check for the existence of logo_email.svg, and fallback to logo_email.gif. That way, if we ever can use SVG in email, we have the code in place. Or if someone doesn't care about GMail support... It shouldn't affect anything negatively, since I would guess nobody already has a file named logo_email.svg in their source.

Reply:
mmm I think it's easier to reintroduce it in the future if there will be the support, it's literally a couple of lines